assets/js/googlesitekit/modules/create-module-store.js

```diff
@@ -68,16 +73,25 @@ export const createModuleStore = ( slug, {
 		storeName,
 	} );
 
+	const infoStore = createInfoStore( {
```

Comment:
See other comment, let's pass `slug` here as first parameter.

Reply:
Slug added to parameters